azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -201,7 +201,8 @@ private SlaOption parseSlaSetting(String set) throws ScheduleManagerException {
         slaInfo.put(SlaOption.ALERT_TYPE, "email");
       }
       if (killAction.equals("true")) {
-        slaActions.add(SlaOption.ACTION_CANCEL_FLOW);
```

Comment:
* I don't see UI change in this PR. WIll there be a separate one? 
* I am confused why id could be checked here? Since Line 207 also check `id`,